Add check for missing `f` prefix for f-strings.

plinss opened the pull request:
Fixes #1

MichaelKim0407 commented on flake8_use_fstring/prefix.py:
```diff
@@ -0,0 +1,80 @@
+        if FSTRING_REGEX.search(token.string):  # already is an f-string
+            return False
+
+        # look ahead for % or .format and skip if present
```
We probably can use an option to toggle this behavior? But, I need to think this through and cover all different use cases.

plinss replied:
I still don't see the value of ignoring that check, but I went ahead and added the option to disable it. I left it on by default.

plinss commented on README.md:
```diff
@@ -19,9 +19,11 @@ pip install flake8-use-fstring
 ## Available Configurations
 
-### `--percent-greedy` and `--format-greedy`
+### `--percent-greedy`, `--format-greedy`, and `--fstring-missing-prefix`
```
Ok, I figured out how to do that, you add `off_by_default = True` to the plugin class.

plinss commented on flake8_use_fstring/base.py:
```diff
@@ -1,5 +1,5 @@
 import token as _token
-import typing as _typing
+from typing import Iterator, List, Tuple
```
Ok, I mostly made that change to get the line to fit in your max length requirement after fixing the return type. Added the '_' to the imports.

plinss commented on flake8_use_fstring/prefix.py:
```diff
@@ -0,0 +1,80 @@
+    r'^[a-zA-Z]*(?:\'\'\'|\'|"""|")(.*?{.+?}.*)(?:\'|\'\'\'|"|""")$')
+
+
+class MissingPrefixDetector(_Base):
```
I split the base classes into a generic one and one that add the greedy handling.

plinss commented:
Ok, I think I addressed all your concerns. Take your time deciding whether or not to include this, I mostly wrote it for my own safety checks, but felt it would be worth contributing back. 

I actually debated if this should be a completely separate plugin, but I felt that functionality fit with responding to the errors generated here.

If nothing else, I'm planning on make a few flake8 extensions myself, so this was a good learning experience.
